src/app/global.css

```diff
@@ -62,4 +62,25 @@ ul {
 #__next {
   position: relative;
   z-index: 0;
+}
+
+.ockConnectWallet_Container span {
+  color: #030712;
+}
+
+/* i believe this is necessary because 
+of a div added by rainbowkit provider */
+body > div {
```

Comment:
I can see junior developer tripping on this.

Can we do this in a more simple way?

Reply:
hmmm i couldn't really think of anything better because it is a hidden div without a classname. do you have an idea?

Reply:
so are you trying to target what in particular? the first `div` after the `body` ?